app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
+  rolify
```

Comment:
I know this is part of the last PR, but does `dfe_sign_in_user` have to be a `attr_writer` ? Could it be an `attr_reader` ?

Reply:
I will talk to you on this at some point. OK for now, as this is still evolving.